package com.revature.registrar.exceptions;

public class DataSourceException extends RuntimeException {
    public DataSourceException(String message, Throwable cause) {
        super(message, cause);
    }
}
